src/main/java/helpers/Utilities.java

```diff
@@ -14,6 +14,7 @@
  */
 public class Utilities {
   public static final String APP_FILE = "./app.properties";
+  public static final String PRODUCT_FILE = "./products.properties";
```

Comment:
This file is being used only in the ProductAddTest. You can move it (along with its getPropertyFromRegProp()) in the test class. Here, in the Utilities class we implement only what is used in multiple places.
So, since the PRODUCT_FILE is used/will be used only by the ProductAddTest you can move it there.

Reply:
done

Reply:
This comment will be address on ticket #50 as well